cmd/cartesi-rollups-node/main.go

```diff
@@ -43,6 +45,9 @@ func main() {
 	slog.SetDefault(logger)
 	slog.Info("Starting the Cartesi Rollups Node", "version", buildVersion, "config", config)
 
+	// run database migrations
+	repository.RunMigrations(fmt.Sprintf("%v?sslmode=disable", config.PostgresEndpoint.Value))
```

Comment:
This won't result in an error if the node is restarted after running for a while, right?

Reply:
No, it will not. Migrations setup in a way that they do not run if they already exist.

It will raise an "error" in the sense that it will not run and warn that migrations have already been set.